integration/client_test.go

```diff
@@ -263,7 +263,7 @@ func (e *Env) setupHistory(ctx context.Context, userID string, signers []signatu
 	// Note that profile 5 is submitted twice by the user to test that
 	// filtering case.
 	for i, p := range [][]byte{
-		nil, nil, cp(1), cp(2), nil, nil, cp(3), nil,
+		nil, cp(1), cp(2), nil, nil, cp(3), nil,
```

Comment:
What is the reason for removing this `nil`? 

Also, if you keep it like this, update the comment above to:
```golang
// following profiles:
// [nil, 1, 2, 2, 2, 3, 3, 4, 5, 5, 5, 5, 5, 5, 6, 6, 5, 7, 7].
```